hugo/content/blog/2020-12-24-mumble-1-4-0-snapshot.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Mumble 1.4.0 Snapshot Announcement
+author: Robert Adam
+date: 2020-12-24
+categories:
+  - News
+  - Snapshot
+---
+
+This year we have a Christmas present for you: We are proud to announce that the first **snapshot release of Mumble 1.4.0** is released as of now. It
+is a very important step towards the actual release of version 1.4.0.
+
+After having understood what it means for a release to be a _snapshot release_ (see below), you can download the new version from our [**Downloads
+page**]({{< relref "/downloads" >}}).
+
+<!--more-->
+
+## Snapshot releases
+
+A _snapshot release_ can be considered an open-beta for an upcoming feature release. This means that it contains the latest and greatest features that
+were introduced since the last feature release. This does also mean that these features have not been tested by a broad audience yet and are
+therefore likely to still contain bugs. These could be minor ones, but in theory these could also be major. Therefore it is important that you always
+back up your data (e.g. configuration files) if you have a previous version of Mumble installed.
+
+Due to the reasoning stated above, we do not recommend using snapshots in an environment where stability of the software is key (e.g. in a business
+context).
```

Comment:
I think we should also mention the target audience. Tech-savy people who are confident in using them.

A “normal” end-user may not be the best audience for this.